packages/cc/src/cc/ZWaveLRProtocolCC.ts

```diff
@@ -0,0 +1,238 @@
+import {
```

Comment:
While this command class is defined in the specs, It's not exposed that I saw in any command class list, so I'm not sure where/how this would be used. Maybe if you had a secondary LR controller in the network? Unknown. We may not need to include this at all.